### ISSUE TITLE
MAP 0.0000

### ISSUE THREAD

**Issue by caihaunqai** [eriklindernoren/PyTorch-YOLOv3#687]
Hello, I am a new guy in objection detection.  I have followed what said in Readme.md and trained custom dataset, but it got map 0.000. I didn't use pretrained model. Do I have anything dismiss? I process the coordinates using these codes, are they true ? Can you help me? Thank you very much!
```
def convert(size, box):
    dw = 1. / (size[0])
    dh = 1. / (size[1])
    x = box[0] + box[2] / 2.0
    y = box[1] + box[3] / 2.0
    w = box[2]
    h = box[3]
 
    x = x * dw
    w = w * dw
    y = y * dh
    h = h * dh
    return (x, y, w, h)

```

**Comment by Flova** [comment]
First of all please use pre-trained weights as they speed up the training by a few days on the large model and a large dataset.

I also don't know the definition of the values in your `box` array. It's therefore hard to say if they are correct.

**Comment by caihaunqai** [comment]
> First of all please use pre-trained weights as they speed up the training by a few days on the large model and a large dataset.
> 
> I also don't know the definition of the values in your `box` array. It's therefore hard to say if they are correct.

Thank you for your reply. I check the data preprocessing, they are true. However, I got Map 0 after 130 epoched. Maybe I am wrong in somewhere? Can you give me any advice?

**Comment by caihaunqai** [comment]
Is this yolov3 contribution really work? Maybe I have to give up~~

**Comment by Flova** [comment]
Maybe give it a try with other hyperparameters such as a lower learning rate.

**Comment by caihaunqai** [comment]
Use another git project, it works. Maybe some  default hyperparameters affects. Thank you for your advice~
